.gitignore

```diff
@@ -18,3 +18,6 @@ dist/
 # Dependency directories (remove the comment below to include it)
 # vendor/
 /.pre-commit-config.yaml
+
+# Others
```

Comment:
do we need changes to this file?

Reply:
We do not. IDE-specific ignores should probably go in that person's global gitignore.